index.html

```diff
@@ -42,10 +42,10 @@ <h2 class="headline">
       <div class="social-handles">
           <p>Let's Connect</p>
           <ul class="social-icons">
-              <li class="github"><a href="https://github.com/ishpaul777"><img src="./assets/icons/github.svg" alt="github"></a></li>
-              <li class="linkedin"><a href="https://www.linkedin.com/in/ishpaul-singh-264590226/"><img src="./assets/icons/linkedin.svg" alt="linkedin"></a></li>
-              <li class="twitter"><a href="https://twitter.com/ishpaul_777"><img src="./assets/icons/twitter.svg" alt="twitter"></a></li>
-              <li class="instagram"><a href="https://www.instagram.com/ishpaul_777"><img src="./assets/icons/instagram.svg" alt=""></a></li>
+              <li class="github"><a href="https://github.com/ishpaul777"><img src="./assets/icons/github.svg" alt="github-icon"></a></li>
+              <li class="linkedin"><a href="https://www.linkedin.com/in/ishpaul-singh-264590226/"><img src="./assets/icons/linkedin.svg" alt="linkedin-icon"></a></li>
+              <li class="twitter"><a href="https://twitter.com/ishpaul_777"><img src="./assets/icons/twitter.svg" alt="twitter-icon"></a></li>
+              <li class="instagram"><a href="https://www.instagram.com/ishpaul_777"><img src="./assets/icons/instagram.svg" alt="instagram-icon"></a></li>
           </ul>
       </div>
   </div>
```

Comment:
- [ ] please your image `alt` should have a brief description of the image since projects haven't been added yet. for example `Diagram of workflow as describe above`